aikau/src/main/resources/alfresco/lists/views/layouts/Carousel.js

```diff
@@ -223,13 +224,7 @@ define(["dojo/_base/declare",
             }
             else
             {
-               // Calculate a suitable height...
-               // We're going to set the height to be as big as the viewing port allows
-               // from the starting vertical position of the node
-               var position = domGeom.position(this.domNode);
-               var viewPort = win.getBox();
-               this.height = viewPort.h - position.y;
-               this.height += "px";
+               this.height = this.calculateHeight(this.domNode) + "px";
```

Comment:
Appending px is going to be a common use case - worth including within the calculateHeight method? (controlled via a boolean param & only added if calculatedHeight isn't undefined?)